tests/storages_tests/test_storages.py

```diff
@@ -1232,3 +1227,28 @@ def failed_trial_callback(study: "optuna.Study", trial: FrozenTrial) -> None:
         optuna.storages.fail_stale_trials(study)
 
         assert study.trials[0].state is TrialState.FAIL
+
+
+@pytest.mark.parametrize("storage_mode", ["sqlite", "redis"])
+def test_fail_stale_trials_raw(storage_mode: str) -> None:
```

Comment:
What is the purpose of this test?

Reply:
```study``` を作成する際に内部で必ず ```get_storage()``` を呼んでしまうので ```fail_stale_trials()``` を生の ```RDBStorage/RedisStorage``` でテストすることは現状できません。

そこで、苦肉の策として生の ```RDBStorage/RedisStorage``` から ```fail_stale_trials()``` を呼んで、エラーにならないことを確認するレベルのテストがこれです。

「これだったら無い方がマシ」ということであれば削除します。

Reply:
Currently, it is not possible to test `fail_stale_trials()` on raw `RDBStorage/RedisStorage` because `get_storage()` is always called internally when the `study` is created.

So, as a "last resort", here is a test that calls `fail_stale_trials()` from raw `RDBStorage/RedisStorage` to make sure it doesn't fail. If you think you're better off without it, I will remove it.

Reply:
Thank you for the explanation. How about adding the comment to note that we should remove this test if we have a option of the `get_storage` to enable a cache?

Reply:
Ok, fixed.